service/src/test/java/greencity/service/HabitAssignServiceImplTest.java

```diff
@@ -289,10 +289,24 @@ void unenrollHabitThrowWrongIdException() {
     @Test
     void deleteHabitAssign() {
```

Comment:
Please rename this method according to the functionality it does
And change Java docs